### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by tacaswell**
work from updating the website during the 1.5.1 release.  These changes are currently live on matplotlib.org, but I forgot to push them.

### CODE REVIEW

**Comment by jenshnielsen** [comment]
The pep8 failure has been fixed on 1.5.x but cannot be fixed on 1.5.1-doc so this looks fine

File: doc/users/github_stats.rst [hunk 1]
```diff
@@ -3,13 +3,13 @@
 Github Stats
 ============
 
-GitHub stats for 2014/08/26 - 2015/10/29 (tag: v1.4.0)
+GitHub stats for 2014/08/26 - 2015/01/09 (tag: v1.4.0)
```
Comment (jenshnielsen) [inline doc/users/github_stats.rst]: Shouldn't this be 2016/01/09?

Comment (tacaswell) [inline doc/users/github_stats.rst]: Yes

**Comment by tacaswell** [comment]
Also fixed the live version: https://github.com/matplotlib/matplotlib.github.com/commit/22cda4197e96512c98887d97f7cf65812738532c

**Pull request merged**